src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -38,6 +38,11 @@
             az managedapp definition create -g MyResourceGroup -n MyManagedAppDef -l eastus --display-name "MyManagedAppDef" \\
                 --description "My Managed App Def description" -a "myPrincipalId:myRoleId" --lock-level None \\
                 --package-file-uri "https://path/to/myPackage.zip"
+        - name: Create a managed application defintion with inline values for createUiDefinition and mainTemplate.
+          text: >
+            az managedapp definition create -g MyResourceGroup -n MyManagedAppDef -l eastus --display-name "MyManagedAppDef" \\
+                --description "My Managed App Def description" -a "myPrincipalId:myRoleId" --lock-level None \\
+                --create-ui-definition @myCreateUiDef.json --main-template @myMainTemplate.json
```

Comment:
Should add `min_profile: latest` to this example so it doesn't show on older API versions as I assume it doesn't work on older API versions.